lib/post.js

```diff
@@ -40,16 +40,16 @@ function postXPI(manifest, options) {
           reject(e);
         });
         client.on("close", function(hadError) {
-          console.log(hadError ? "Posting XPI to %s failed" :
+          console.log(hadError ? "Posting XPI to %s failed" : // eslint-disable-line no-console
```

Comment:
This looks a bit weird since the `console.log()` is wrapped on 2 lines, but I had to put the comment on the same line as opening `console.log()`.